### PR TITLE
Disable cors on vod api

### PR DESCRIPTION
**Pull request by gioelecerati**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

## -

- **How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/api/src/controllers/asset.ts [hunk 1]
```diff
@@ -130,6 +130,7 @@ const fieldsMap: FieldsMap = {
 };
 
 app.get("/", authMiddleware({}), async (req, res) => {
+  res.header("Access-Control-Allow-Origin", "*");
```
Comment (victorges) [inline packages/api/src/controllers/asset.ts]: You should look into the cors plugin/library that we use somewhere in this repo. Should be in the places where we set up the whole express application like app-router or something like that

Comment (gioelecerati) [reply packages/api/src/controllers/asset.ts]: Wouldn't adding the wildcard on the cors definition in `index.ts` add it to all the endpoints and not the VOD ones? I saw it was specifically added on some endpoints (e.g. the stream proxy)

Comment (gioelecerati) [reply packages/api/src/controllers/asset.ts]: NVM, added only on vod via lib

Comment (iameli) [reply packages/api/src/controllers/asset.ts]: You need access-control-allow-method too

Comment (gioelecerati) [reply packages/api/src/controllers/asset.ts]: added as well

**Comment by iameli** [comment]
LGTM! go ahead and merge.

**Pull request merged**